pkgs/applications/networking/feedreaders/feeds/default.nix

```diff
@@ -0,0 +1,112 @@
+{ lib
+, stdenv
+, fetchFromGitLab
+
+, appstream
+, gobject-introspection
+, meson
+, ninja
+, pkg-config
+, wrapGAppsHook
+
+, glib
+, gtk3
+, libhandy_0
+, webkitgtk
+, python3
+}:
+let
+  listparser = python3.pkgs.buildPythonPackage rec {
+    pname = "listparser";
+    version = "0.18";
+
+    src = python3.pkgs.fetchPypi {
+      inherit pname version;
+      sha256 = "0hdqs1mmayw1r8yla43hgb4d9y3zqs5483vgf8j9ygczkd2wrq2b";
```

Comment:
One issue might be that r-ryantm might not update files with multiple hashes in them.